test/integration/test_load_study.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# exit when any of these fails
+set -e
+
+run_in_service() {
+    service=$1
+    shift
+    docker-compose -f docker-compose.yml -f ../test/integration/docker-compose-localbuild.yml \
+        run --rm \
+        "$service" bash -c "$@"
+}
+
+# load panels
+run_in_service cbioportal "cd /cbioportal/core/src/main/scripts/ && perl importGenePanel.pl --data \
+                          /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel1.txt"
+run_in_service cbioportal "cd /cbioportal/core/src/main/scripts/ && perl importGenePanel.pl --data \
+                          /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel2.txt"
+
+# dump portal info
```

Comment:
Now, you only test the route via portaldump, isn't it? I think it is a good idea to also add the study load option that uses the server url (-u option) so that validation is against a live portal, not the dump of the portal. Or is such a test redundant?